notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,588 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9de9a93e-9247-4799-a5bb-2ec1575ae8c2",
+   "metadata": {},
+   "source": [
+    "# Live 3D Human Pose Estimation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates a live 3D Human Pose Estimation webcam example with OpenVINO. We utilize the model [human-pose-estimation-3d-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/human-pose-estimation-3d-0001) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). At the last session of this notebook, you will see the live inference results from your webcam (if available). Alternatively, you can also upload a video file to test out the algorithms.\n",
+    "**Make sure you have properly installed the [Jupyter extension](https://github.com/jupyter-widgets/pythreejs#jupyterlab) and using Jupyterlab to run the demo as mentioned in the README.md**\n",
+    "\n",
+    "> **NOTE**: _To use the webcam, you must run this Jupyter notebook on a computer with a webcam. If you run on a server, the webcam will not work. However, you can still do inference on a video in the final step. This demo utilizes the python interface of Three.js integrated with webgl to process data derived from model inference. And it displays the processed result in the notebook._\n",
+    "\n",
+    "_To ensure that the results are displayed correctly, we recommend that you run the code in a browser on the following operating system:_  \n",
+    "_Ubuntu, Windows: chrome_  \n",
+    "_MacOS: Safari_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7925a51b-26ec-43c5-9660-0705c03d724d",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "**The Pythreejs extension may not display properly under the Jupyter Notebook for its latest release (2.4.1), so it is recommended to use the Jupyter Lab.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b84c1f5e-502b-4037-b871-9f84b4e8cef0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -r requirements.txt"
```

Comment:
Let's use pythreejs directly here instead of requirements.txt. This is just one package :)